TundraTD/Assets/Scripts/Mobs/MobModel.cs

```diff
@@ -26,7 +26,7 @@ public class MobModel : MonoBehaviour
         [SerializeField]
         private SkinnedMeshRenderer skinRenderer;
 
-        [SerializeField] private Rigidbody rigidbody;
+        [SerializeField] new private Rigidbody rigidbody;
```

Comment:
I guess there could be something incorrect when using `new` modifier. Maybe there were a variable with the same purpose? If no, you could rename it

Reply:
Rigidbody is a Obsolete and unused method for all monobehaviours. The use of 'rigitbody' as it is named there is necessary